src/Patterns/Command/CommandCache.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Neuron\Patterns\Command;
+
+class CommandCache implements ICommandCache
+{
+	/**
+	 * @var ICommandCache|null
+	 */
+	private static ?ICommandCache $_Instance = null;
+
+	/**
+	 * @var string[]
+	 */
+	private static array $_Cache = [];
+
+	/**
+	 * Making the constructor private to not allow to get the class instance using the "new" keyword
+	 */
+	private function __construct()
+	{
+	}
+
+	/**
+	 * @return ICommandCache
+	 */
+	public static function getInstance(): ICommandCache
```

Comment:
Why is there duplicate singleton code here?

Reply:
@ljonesfl I have updated the PR, please review updates